### DISCUSSION TITLE
Adding more images to a task

### DISCUSSION THREAD

**Discussion by bilalcodehub** [Project-MONAI/MONAILabel#1294] [General]
Hi,

I am using the following command to start `monailabel` server for annotating endoscopic data:
```
monailabel start_server --app apps/endoscopy --studies datasets/images_short --conf models tooltracking --conf epistemic_enabled true --conf epistemic_top_k 10
```
I get one task `ActiveLearning_Iteration_1` with ten jobs each having a single image as shown below:

<img width="1009" alt="image" src="https://user-images.githubusercontent.com/7869659/218223047-dc52d184-8cee-45ff-8ab6-6c634f173d73.png">

I was expecting `monailabel` to create one job with ten images but that is not what happened. It created ten jobs with one image each. Now I have to score each image mark it complete and then go to another task. It is not efficient if I need to annotate thousands of images.

How can I tell `monailabel` to create jobs with k number of images?

**Comment by tangy5** [comment]
HI @bilalcodehub , thanks for using the MONAI Label, in the endoscopy+CVAT integration: this is the case currently, each image is associated with one job with MONAI Label, so top_k is the number of images/jobs it will upload. 

If you would like to customize the CVAT datastore and upload logics and familiar with CVAT software. You could refer to these script to customize your demand. 

https://github.com/Project-MONAI/MONAILabel/blob/main/sample-apps/endoscopy/lib/scoring/cvat.py

https://github.com/Project-MONAI/MONAILabel/blob/main/monailabel/datastore/cvat.py

**Reply by SachidanandAlle** [reply]
There is an option to specify number of frames per task.. it's mentioned on the GitHub readme.. 

cvat_segment_size

That will help you to control how many frames per task u want to push

**Reply by bilalcodehub** [reply]
Thanks @tangy5 and @SachidanandAlle for the support.

I'm a newbie in `CVAT` and `MONAILabel`. Used `Prodigy` mostly for labelling. `MONAILabel` excites me due to its enormous potential.

@SachidanandAlle: I tried `cvat_segment_size` it worked. I used the following command:
```
monailabel start_server --app apps/endoscopy --studies datasets/images_short --conf models tooltracking --conf epistemic_enabled true --conf epistemic_top_k 10 --conf cvat_segment_size 10
```
This is allowing `MONAILabel` to create one job with a batch of ten images, as shown below.
<img width="1016" alt="image" src="https://user-images.githubusercontent.com/7869659/218252969-c0628bfc-537a-4bf4-8030-d52b6410bb13.png">


Thanks a lot.
